packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -183,7 +183,7 @@
             during my use of the Free Trial Service and Okta’s
             <SmartLink :item="{ link: 'https://www.okta.com/privacy-policy' }"
               >Privacy Policy</SmartLink
-            >. I further agree that Okta may contact me with marketing
+            >.<br /> I further agree that Okta may contact me with marketing
```

Comment:
Could this actually be in a separate `p` tag? We wouldn't need to put a `<br />` tag and it will also would hopefully provide better spacing.